taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -1577,6 +1587,67 @@ export class VsCodeHelper {
 		});
 	}
 
+	exposeInvokeEntrypointCommand() {
+		this.registerCommand(
+			'invoke_entrypoint',
+			async (item: SmartContractEntrypointTreeItem) => {
+				const michelineParameters = item.michelineParameters;
+				const panel = this.vscode.window.createWebviewPanel(
+					'entrypointParameter',
+					`Editing ${item.name} parameters`,
+					this.vscode.ViewColumn.One,
+					{
+						enableScripts: true,
+					},
+				);
+				const result = createVscodeWebUiHtml({
+					webview: panel.webview,
+					subscriptions: this.context.subscriptions,
+					interop: {
+						view: 'MichelineEditor',
+						input: {
+							dataType: michelineParameters,
+							actionTitle: 'Invoke Entrypoint',
+						},
+						onMessage: async x => {
+							if (x.kind === 'action') {
+								const fileName = 'invoke_temp_file.tz';
+								fs.writeFileSync(
```

Comment:
As this is an async function, can we use `await fs.writeFile` instead?

Reply:
The `fs.writeFile` does not return a `Promise` to await. We should pass a callback to it.